addons/share-through-mystuff/userstyle.css

```diff
@@ -0,0 +1,16 @@
+.media-item-content.not-shared .media-action {
+  top: 20px !important;
```

Comment:
`top` is set to `0px` lower down in the file, so maybe that's why you used `!important` here? If so could you remove this and set it to 20px down there?

Reply:
Honestly, I have no idea why that's there. After a bit of testing it seems that all it does is merge the delete and share buttons, making them impossible to read, let alone click. Here's what it looks like for me if I remove both of them:
![Screenshot from 2023-01-20 19-39-52](https://user-images.githubusercontent.com/75220768/213841923-94c7f0b3-72be-4ecc-ac1f-00c68618ee6a.png)

Reply:
I can't get it to work without both(as in both buttons are layered on top of each other without both). Not sure what's up with that, or if I'm doing something wrong.

Reply:
I still don't think `!important` is necessary, but I could be wrong.

Reply:
I think you're right, I'll remove `!important`.